### PR TITLE
Update FE README & Package.json

### LINKED ISSUE #104: Update ReadMe for frontend
update the fronend readme

### PR DESCRIPTION
**Pull request by Marie-Ehrman**
Created FE README.

Also react-scripts was causing an error on version 3.3.0. running `npm install react-scripts@latest` installed and updated all necessary files and resolved the issue!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Dwellingly Frontend
+
+This serves as the Frontend of the Full Stack Application for [JOIN Organization's](https://joinpdx.org/) Dwellingly site being created per the FIGMA designs. This site will eventually aid in supporting both tenants
```
Comment (Chris-Boe) [inline README.md]: Should we put a link to the Figma PDF here?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+From Terminal:
+
+- Navigate to your desired directory from the command line:
+    example:`cd Desktop/CodeForPDX/Dwellingly`
+- Clone Project - `git clone https://github.com/codeforpdx/dwellingly-app.git`
+- Create your own working branch:
+    example: `git checkout -b add-new-component`
+
+
+### Installing
```
Comment (Chris-Boe) [inline README.md]: Let's also add a link to the dwellinglybackend project to tell newcomers where to look for the API project and its separate Readme

**Review by Chris-Boe — CHANGES_REQUESTED** [review]
Nice job! Just a couple ideas for more info

File: README.md (hunk 1 above)
Comment (Marie-Ehrman) [reply README.md]: Absolutely! I had it originally and wasn't sure if I should.

File: README.md (hunk 2 above)
Comment (Marie-Ehrman) [reply README.md]: YES! I had this on my list and completely forgot to do it before my PR yesterday! Thanks, @Chris-Boe!

**Review by Chris-Boe — APPROVED** [review]

**Pull request merged**